bin/sce_qc_report.R

```diff
@@ -224,6 +230,7 @@ scpcaTools::generate_qc_report(
   filtered_sce = filtered_sce,
   processed_sce = processed_sce,
   report_template = opt$report_template,
-  output = opt$qc_report_file
+  output = opt$qc_report_file,
+  seed = opt$seed
```

Comment:
Implementing my suggestion in https://github.com/AlexsLemonade/scpca-nf/pull/432#issuecomment-1708955195
```suggestion
  extra_params = list(seed = opt$seed)
```